### ISSUE TITLE
Bug with allow only-planned tasks (#234)

### ISSUE THREAD

**Issue by agarciacalvo** [jsGanttImproved/jsgantt-improved#251]
**Describe the bug**
When adding a task with only plan dates, the grid is showing real dates.

See https://github.com/jsGanttImproved/jsgantt-improved/issues/234#issuecomment-515657931

**Expected behavior**
The grid should show empty real dates.

**Screenshots**
![image](https://user-images.githubusercontent.com/18212145/64023160-f8d72580-cb37-11e9-817b-3e115c50d454.png)

**Comment by mariohmol** [comment]
Indeed... i guess we had this fix in the past.. maybe we did for plan?

**Comment by agarciacalvo** [comment]
I don't know for sure. After #234 the component allowed only planned tasks and they are showing well in the gantt (with proper transparency), but with real dates in the grid. As we can see in the fourth task of the screenshot, we can have one task without plan end date, so perhaps it has been one bug on that fixed in the past, but I think this is more one pending polishing on #234 that I don't know how to address.

**Comment by mariohmol** [comment]
You are right.. Thanks!

**Comment by agarciacalvo** [comment]
Thank you so much for merging the pull request. Now this is fixed. However, I'm opening a new issue due that this is not fixing task group dates.
